style.css

```diff
@@ -1 +1,100 @@
+body {
+  width: 1200px;
+}
 
+header {
+  width: 900px;
+  height: 50px;
+  margin: 0px auto;
```

Comment:
Instead of applying a width and `margin: 0 auto;` to each of the elements you want centered on the page, it is simpler to set the width and margin on the `body` or some wrapper and have all the elements be full width inside of it.